image_recognition_util/scripts/colour_extractor_node

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env python
```

Comment:
@JosjaG what to do with this given @reinzor's much more elaborate (but less functional? I know this thing here works :-) ) color_extractor. 
That uses the Recognition API as well, ths integrating more tight with the other tools.

Reply:
We could compare the color_extractors to see if the new one is indeed better. If it is, we could also check if the new color_extractor could be integrated in the original one. Might be too much effort before RWC2019 though.

Reply:
Why is this in the util package? Util package was created as library, not a package holding nodes.